src/TLuaInterpreter.h

```diff
@@ -233,7 +233,7 @@ Q_OBJECT
     static int Send( lua_State * L );
     static int sendRaw( lua_State * L );
     static int Echo( lua_State * L );
-    static int select( lua_State * L );
+    static int selectString( lua_State * L ); // Was select but I think it clashes with the Lua command with that name
```

Comment:
There is plenty of scope for the naming here and in the external Lua files leading to confusion, e.g. see the `Echo` above which is not the same as `echo` to the user... :see_no_evil: